src/app/(pages)/pool/[pool-id]/_components/pool-details.tsx

```diff
@@ -39,8 +40,8 @@ export default async function PoolDetails({ pool }: { pool: PoolDetailsDTO }) {
                     {pool.status != POOLSTATUS.ENDED && (
                         <PoolDetailsProgress
                             data-testid='pool-details-progress'
-                            current={pool.poolBalance}
-                            goal={pool.goal}
+                            current={Number(formatUnits(BigInt(pool.poolBalance), pool.tokenDecimals))}
```

Comment:
Why do we need to convert to Bigint, then parsing, then converting again to number? This specific part was working perfectly and it is completely unrelated to the participants tabs, can you elaborate on this change?

Reply:
I was using the same component and noticed there was the token decimal parsing issue. This is what I'm seeing on the main branch.
<img width="545" alt="Screenshot 2024-10-24 at 13 19 02" src="https://github.com/user-attachments/assets/cfd186e7-183e-4c10-a680-4b8d25c10058">

Reply:
Good catch, Jonathan and I also identified it and I am working on a fix. Thanks for pointing out!